validator/validator-main.protoascii

```diff
@@ -3521,6 +3521,7 @@ tags: {  # <amp-img>
     supported_layouts: FLEX_ITEM
     supported_layouts: NODISPLAY
     supported_layouts: RESPONSIVE
+    supported_layouts: INTRINSIC
```

Comment:
`amp-layout` rule just below needs this too

Reply:
adding

Reply:
alphabetize `supported_layouts`.